test/flows/uk_benefits_abroad_flow_test.rb

```diff
@@ -0,0 +1,1302 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class UkBenefitsAbroadFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup do
+    testing_flow UkBenefitsAbroadFlow
+    stub_worldwide_api_has_locations(
+      %w[
+        australia
+        barbados
+        bermuda
+        canada
+        gibraltar
+        guernsey
+        ireland
+        israel
+        jamaica
+        jersey
+        liechtenstein
+        mauritius
+        new-zealand
+        north-macedonia
+        turkey
+        usa
+      ],
+    )
+  end
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: going_or_already_abroad?" do
+    setup { testing_node :going_or_already_abroad? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of which_benefit? for any response" do
+        assert_next_node :which_benefit?, for_response: "going_abroad"
+      end
+    end
+  end
+
+  context "question: which_benefit" do
```

Comment:
```suggestion
  context "question: which_benefit?" do
```

Teeny consistency issue